tests/integration/converters/test_ome.py

```diff
@@ -25,32 +25,39 @@ def test_ome(format_path):
     # os_img = osld.open_slide(get_path("CMU-1-Small-Region.svs.tiff"))
 
     t = TileDBOpenSlide.from_group_uri(get_path(format_path))
-    dset = CMU_1_SMALL_REGION()
-    schemas = dset.schema()
+    schemas = CMU_1_SMALL_REGION().schemas()
+
+    for i in range(0, 3):
+        assert t.level_info[i] == LevelInfo(
+            uri="", level=i, dimensions=schemas[i].shape
+        )
 
-    assert (
-        t.level_info[0] == LevelInfo(uri="", level=0, dimensions=schemas[0].shape)
-        and t.level_info[1] == LevelInfo(uri="", level=1, dimensions=schemas[1].shape)
-        and t.level_info[2] == LevelInfo(uri="", level=2, dimensions=schemas[2].shape)
-    )
     assert t.level_count == 3
     assert t.dimensions == (2220, 2967)
     assert t.level_dimensions == ((2220, 2967), (387, 463), (1280, 431))
     assert t.level_downsamples == ()
 
 
-def test_ome_tiff_converter(tmp_path):
-    expected = CMU_1_SMALL_REGION().schema()
-    src = get_path("CMU-1-Small-Region.ome.tiff")
-    dest = os.path.join(tmp_path, ".tiledb")
-    if not os.path.exists(dest):
-        OMETiffConverter().convert_image(src, dest, level_min=0)
-    with tiledb.open(os.path.join(dest, "l_0.tdb")) as A:
-        assert A.schema == expected[0]
-    with tiledb.open(os.path.join(dest, "l_1.tdb")) as A:
-        assert A.schema == expected[1]
-    with tiledb.open(os.path.join(dest, "l_2.tdb")) as A:
-        assert A.schema == expected[2]
+@pytest.mark.parametrize(
+    "converter",
```

Comment:
[parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) accepts comma separated params, e.g. "converter,path" which avoids the need for `converter[0]/converter[1]`